app/components/Schedule/SessionCategories.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+    <ul class="categories-list">
+        <li
+            v-for="(category, j) in categories"
+            :key="`category-${listId}-${j}`"
+            :style="getCategoryStyle(category.colors)"
+            class="category-item"
+        >
+            {{ category.name }}
+        </li>
+    </ul>
+</template>
+
+<script setup>
+const props = defineProps({
+    listId: {
+        type: String,
+        required: true,
+    },
+    categories: {
+        type: Array,
+        required: true,
+    },
+    hoverColors: {
+        type: Object,
+        default: null,
+    },
+});
+
+const getCategoryStyle = (colors) => {
+    return {
+        '--category-bg': colors.bg,
+        '--category-text': colors.text,
+        '--hover-text': props.hoverColors?.['--hover-bg'] || colors.bg,
+        '--hover-bg': props.hoverColors?.['--hover-text'] || colors.text,
```

Comment:
Tu peux ajouter une default value à une variable CSS, c'est bien plus clean comme ça 
```suggestion
        '--hover-text': var(--hover-bg, colors.bg)
        '--hover-bg': var(--hover-text, colors.text)
```

Reply:
J'aimerais bien mais ça fit pas dans ce contexte